app/notifications/apns.py

```diff
@@ -0,0 +1,85 @@
+from app.server import server
+
+from hyper import HTTPConnection
+from config import notifications
+from jwcrypto.jwt import JWT
+from jwcrypto.jwk import JWK
+from datetime import timedelta
+from time import time
+from os import path, getcwd
+from json import dumps as json_dumps, loads as json_loads
+
+import bugsnag
+
+"""
+Communicates with APNS.
+"""
+
+apns_key = 'apns.p8'
+apns_expiration = timedelta(days=1).total_seconds()
```

Comment:
Though this isn't a true constant thanks to DST and leap seconds, it's used as one, so it should be all-caps